templates/plugin/dhcp_remote_isc.yml.erb

```diff
@@ -0,0 +1,30 @@
+---
+#
+# Configuration file for remote ISC dhcp provider
+#
+
+:config: <%= scope.lookupvar("::foreman_proxy::plugin::dhcp::remote_isc::dhcp_config") %>
```

Comment:
Since the template is rendered inside the remove_isc class, all these variables can be access through the local scope (```@dhcp_config```).